pdal/QuickInfo.hpp

```diff
@@ -34,6 +34,9 @@
 
 #pragma once
 
+#include <pdal/JsonFwd.hpp>
+#include <nlohmann/json.hpp>
```

Comment:
I think we just want the fwd here and the implementation in the QuickInfo.cpp file.